ledger/acctupdates.go

```diff
@@ -1625,14 +1625,6 @@ func (au *accountUpdates) prepareCommit(dcc *deferredCommitContext) error {
 	// verify version correctness : all the entries in the au.versions[1:offset+1] should have the *same* version, and the committedUpTo should be enforcing that.
 	if au.versions[1] != au.versions[offset] {
 		au.accountsMu.RUnlock()
-
```

Comment:
this code is the same as catchpointtracker's `handleUnorderedCommitOrError` and called after commitRound / prepareCommit for all trackers in case of errors.